src/main/resources/db/changesets/20181203_studytable.yaml

```diff
@@ -0,0 +1,208 @@
+                  name: from_column
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_study_relationship_study_column_from
+                    references: study_column(id)
+              - column:
+                  name: to_column
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_study_relationship_study_column_to
+                    references: study_column(id)
+
+        - createTable:
+            tableName: asset_specification
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: study_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_asset_specification_study
+                    references: study(id)
+              - column:
+                  name: name
+                  type: varchar(256)
+
+        - createTable:
+            tableName: asset_table
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: asset_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_asset_table_asset_specification
+                    references: asset_specification(id)
+              - column:
+                  name: table_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_asset_table_study_table
+                    references: study_table(id)
+              - column:
+                  name: is_root
+                  type: boolean
+
+        - createTable:
+            tableName: asset_column
+            columns:
+              - column:
+                  name: id
+                  type: ${uuid_type}
+                  defaultValueComputed: ${uuid_function}
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: asset_id
+                  type: ${uuid_type}
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_asset_column_asset_specification
+                    references: asset_specification(id)
+              - column:
+                  name: column_id
```

Comment:
There are a lot of column_ids. It might help disambiguate if references to other column ids are prefixed; e.g., study_column_id. Maybe...

Reply:
I was considering that this morning. I agree, will fully qualify.